pkg/config/config.go

```diff
@@ -234,7 +234,7 @@ func getBoolConfig(configKey string, defaultBool bool) bool {
 }
 
 func getIntConfig(configKey string, defaultInt int) int {
-	defaultValue := fmt.Sprintf("%d", defaultInt)
+	defaultValue := strconv.Itoa(defaultInt)
```

Comment:
fix `Error: pkg/config/config.go:237:36: defaultInt escapes to heap`
ref https://github.com/sustainable-computing-io/kepler/actions/runs/12177539929/job/33966954289 and https://github.com/sustainable-computing-io/kepler/actions/runs/12197388859/job/34026929446?pr=1886